### PR TITLE
Improve the checker documetations

### PR DESCRIPTION
**Pull request by sbrunner**
Fix: https://jira.camptocamp.com/browse/GSGMF-898

### CODE REVIEW

File: doc/integrator/checker.rst [hunk 1]
```diff
@@ -135,6 +135,7 @@ Uses the global configuration ``available_locale_names``, and the checker config
 Check some routes, configured in ``routes`` as array of objects with::
 
   * ``name`` the route name.
+  * ``display_name`` tha name to be displayed.
```
Comment (jwkaltz) [inline doc/integrator/checker.rst]: the name to be displayed

File: doc/integrator/checker.rst [hunk 2]
```diff
@@ -150,6 +151,17 @@ using the ``routes`` configuration as an array of route names to check::
   * ``params`` the used query string as a dictionary.
   * ``level``
 
+Infrastructure
```
Comment (jwkaltz) [inline doc/integrator/checker.rst]: Suggestion for title:
Application supervision

File: doc/integrator/checker.rst [hunk 3]
```diff
@@ -150,6 +151,17 @@ using the ``routes`` configuration as an array of route names to check::
...
+--------------
+
+We can use the following configurations:
```
Comment (jwkaltz) [inline doc/integrator/checker.rst]: Suggestion:
To be able to verify that your application is operational, you can define a "checker". Configure the checker as follows:

Comment (sbrunner) [reply doc/integrator/checker.rst]: => In the ``checker`` configuration you can add the following configuration to solve some connection issues.

Looks good?

Comment (jwkaltz) [reply doc/integrator/checker.rst]: How about:

If you experience connection issues with your ``checker``, the following configuration options may be useful for you:

Comment (sbrunner) [reply doc/integrator/checker.rst]: +1, Thanks :-)

**Pull request merged**